src/ui/common/src/components/workflows/artifact/content.tsx

```diff
@@ -0,0 +1,110 @@
+import { CircularProgress } from '@mui/material';
+import Alert from '@mui/material/Alert';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+
+import { ArtifactResultResponse } from '../../../handlers/responses/artifact';
+import { ContentWithLoadingStatus } from '../../../reducers/artifactResultContents';
+import { ArtifactType } from '../../../utils/artifacts';
+import { isFailed, isInitial, isLoading } from '../../../utils/shared';
+import { Button } from '../../primitives/Button.styles';
+import PaginatedTable from '../../tables/PaginatedTable';
+
+type Props = {
+  artifact: ArtifactResultResponse;
+  contentWithLoadingStatus?: ContentWithLoadingStatus;
+};
+
+const ArtifactContent: React.FC<Props> = ({
+  artifact,
+  contentWithLoadingStatus,
+}) => {
+  if (!artifact.result) {
+    return (
+      <Typography variant="h5" component="div" marginBottom="8px">
+        No result to show for this artifact.
+      </Typography>
+    );
+  }
+
+  if (artifact.result.content_serialized != null) {
```

Comment:
!==

Reply:
let me add a comment, this is intentional as it makes sure the field is neither `null` nor `undefined`